hypothesis-python/scripts/basic-test.sh

```diff
@@ -17,6 +17,8 @@ PYTEST="python -m pytest -n2"
 
 $PYTEST --runpytest=subprocess tests/pytest
 
+PYTHONOPTIMIZE=2 $PYTEST tests/cover/test_testdecorators.py
```

Comment:
Is there a reason we're only running one test file with PYTHONOPTIMIZE=2? Have we discussed it before, and I've forgotten?

Reply:
I *think* we've discussed it at some point - running the whole suite would be very inefficient given that we've only ever had one bug related to `PYTHONOPTIMIZE`, and running these core tests exercises our API enough to flush out that and any similar bugs.  There's just not much point doing more when the assertions are all gone!

Reply:
Ah, drilling back down the issue list, it's because the original bug was in one of these decorators – see https://github.com/HypothesisWorks/hypothesis/issues/822

Could we get a comment with a combination of what we both just said please? Then I think this’ll be good to go.

Reply:
Done :smile: